src/content/translations/pt-br/developers/tutorials/how-to-mint-an-nft/index.md

```diff
@@ -3,10 +3,9 @@ title: Como criar um NFT (Segunda parte da série de tutoriais sobre NFT)
 description: Este tutorial descreve como criar um NFT na blockchain Ethereum usando nosso contrato inteligente e Web3.
 author: "Sumi Mudgil"
 tags:
-  - "NFTs"
   - "ERC-721"
-  - "Alchemy"
-  - "Solidity"
+  - "alchemy"
```

Comment:
```suggestion
  - "Alchemy"
```

Reply:
```suggestion
```

I think we just get rid of the project name here